node-lib/src/options.rs

```diff
@@ -153,6 +158,10 @@ pub struct RunOptions {
     #[clap(long)]
     pub p2p_max_clock_diff: Option<u64>,
 
+    /// Optional list of whitelisted addresses.
+    #[clap(long, value_name = "ADDR")]
+    pub p2p_whitelist_addr: Option<Vec<IpAddr>>,
+
```

Comment:
Let's put this one right after `p2p_reserved_node`, as it's done in `P2pConfig`.